# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

counter_with_delayActionGoal action_goal
counter_with_delayActionResult action_result
counter_with_delayActionFeedback action_feedback
